deployment/ansible/roles/nyc-trees.app/templates/nginx-nyc-trees-app.conf.j2

```diff
@@ -6,6 +6,20 @@ server {
 
   access_log /var/log/nginx/nyc-trees-app.access.log logstash_json;
 
+  location /favicon.ico {
```

Comment:
You're right. I didn't clear things out first.

Is the `favicon.ico` file in this PR valid? When I try to view it, I get something that looks like a broken image.